Add options to configure .github/workflows/meta.yml (#150)
plone/meta: config/README.md

### `.github/workflows/meta.yml`

Add the `[github]` TOML table in `.meta.toml`,
and set the enabled jobs with the `jobs` key.

```toml
[github]
jobs = [
    "qa",
    "test",
    "coverage",
    "dependencies",

>     "release-ready",

Reviewer: ```suggestion
    "release_ready",
```

Let's use the actual name, so we can simplify the template, there's already way too many (and more to come) options to remember how things should be named.

Maybe we can mention that the complete list is on https://github.com/plone/meta/tree/main/.github/workflows ? 🤔

    "circular",
    ]

> """

Reviewer: ```suggestion
```

Copy & pasted one line too much 😄

```

Extend github workflow configuration with additional jobs
by setting the values for the `additional_jobs` key.

```toml
[github]

> additional_jobs = """

Reviewer: Could we rename that, for consistency across the other files, `extra_lines` ? 🍀 🤞🏾

  another:
    uses: org/repo/.github/workflows/file.yml@main
"""

> ```

Reviewer: Would you mind adding another documentation section here for the `branch` variable?

I would like to eventually start using tags for this `branch` variable, so we can _safely_ evolve the workflows while at the same time ensuring we don't break existing workflows on repositories 🍀 

Maybe we can rename it `ref` ? 🤔

Author: Done

### `.github/workflows/meta.yml`

Add the `[github]` TOML table in `.meta.toml`,
and set the enabled jobs with the `jobs` key.

```toml
[github]
jobs = [
    "qa",
    "test",
    "coverage",
    "dependencies",
    "release_ready",
    "circular",
    ]
```

It is possible to configure from which branch/tag of `plone/meta`
to get the workflow files by setting the value of the `ref` key:

```toml
[github]
ref = "master"
```

In the previous example, all GitHub workflows would come from
the `master` branch, instead of the default `main` branch.

Extend github workflow configuration with additional jobs

> by setting the values for the `additional_jobs` key.

Reviewer: ```suggestion
by setting the values for the `extra_lines` key.
```
